RecoVertex/BeamSpotProducer/test/test_scalars.py

```diff
@@ -17,31 +17,23 @@
 
 process.maxEvents = cms.untracked.PSet(
         input = cms.untracked.int32(-1)
-)        
-process.load("CondCore.DBCommon.CondDBSetup_cfi")
+)
+process.load("CondCore.CondDB.CondDB_cfi")
```

Comment:
tagging here, for lack of a better place, but is the name of this file still sensible?
In Run3 after all we won't be using scalers (there's actually a typo in the file name I guess).
What's the purpose of:
```
process.scalersRawToDigi = cms.EDProducer('ScalersRawToDigi')
```
just asking for my curiosity.

Reply:
They are leftover from old code, thanks for spotting this! I will udpate and rename the test script.